app/components/form/save-with-history.hbs

```diff
@@ -1,30 +1,36 @@
-<AuModal
-  @id="form-history"
-  @closeModal={{fn (mut @isModalOpen) false}}
-  @modalOpen={{@isModalOpen}}
-  class="small"
-  @title="Beschrijving doorgevoerde wijzigingen."
-  as |Modal|
->
-  <Modal.Body>
-    <AuInput
-      @id="history-message"
-      @width="block"
-      placeholder="Beschrijving wijzigingen"
-      {{on "input" @updateValue}}
-    />
-  </Modal.Body>
-  <Modal.Footer>
-    <AuToolbar as |Group|>
-      <Group>
-        <AuButton
-          {{on "click" @save}}
-          @loading={{@isSaving}}
-          @loadingMessage="Aan het opslaan"
-        >
-          Bevestig
-        </AuButton>
-      </Group>
-    </AuToolbar>
-  </Modal.Footer>
-</AuModal>
\ No newline at end of file
+{{#if @isModalOpen}}
+  {{#if @addDescription}}
+    <AuModal
+      @id="form-history"
+      @closeModal={{fn (mut @isModalOpen) false}}
+      @modalOpen={{@isModalOpen}}
+      class="small"
+      @title="Beschrijving doorgevoerde wijzigingen."
+      as |Modal|
+    >
+      <Modal.Body>
+        <AuInput
+          @id="history-message"
+          @width="block"
+          placeholder="Beschrijving wijzigingen"
+          {{on "input" @updateValue}}
+        />
+      </Modal.Body>
+      <Modal.Footer>
+        <AuToolbar as |Group|>
+          <Group>
+            <AuButton
+              {{on "click" @save}}
+              @loading={{@isSaving}}
+              @loadingMessage="Aan het opslaan"
+            >
+              Bevestig
+            </AuButton>
+          </Group>
+        </AuToolbar>
+      </Modal.Footer>
+    </AuModal>
+  {{else}}
+    <code {{did-insert @save}} />
```

Comment:
this doesn't feel quite right. Shouldn't the instance component just do this? and either trigger the saveWithHistory or not depending on @showHistoryDescription?

Reply:
Ill update it that it is being handled  in a higher layer